insecure/fixtures.go

```diff
@@ -0,0 +1,89 @@
+package insecure
```

Comment:
This file does not contain any new changes, just moving some of our fixtures to here.